low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -403,8 +404,9 @@ class SSPLDevDeploy:
             "%s post_install --config yaml://%s" % (utils_setup_script, post_install_config),
             "%s post_install --config yaml://%s" % (sspl_setup_script, post_install_config),
             "%s prepare --config yaml://%s" % (sspl_setup_script, prepare_config),
-            "%s config --config yaml://%s" % (utils_setup_script, config_config),
+            "%s config --config yaml://%s" % (utils_setup_script, utils_config_tmpl),
             "%s config --config yaml://%s" % (sspl_setup_script, config_config),
+            "%s init" % (utils_setup_script),
```

Comment:
why two times init needs to be called, thats not proper. Merge required config in either miniprov Config or Init stage

Reply:
first init is for utils_setup and next init phase is of sspl_setup script